internal/command/extensions/arcjet/arcjet.go

```diff
@@ -0,0 +1,23 @@
+package arcjet
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/superfly/flyctl/internal/command"
+)
+
+func New() (cmd *cobra.Command) {
+
+	const (
+		short = "Protect your applications with Arcjet"
```

Comment:
We should keep this language functional, similar to what we have now:

"Provision and manage Arcjet WAF sites" or similar. We can add a longer description in the long help which should display with `fly ext arcjet`.

Reply:
Note that we want to improve visibility of extensions, but will do that in a separate project we have going on.

Reply:
Let’s use “Provision and manage Arcjet”. We don’t want to use “WAF” because that’s only a small component of what we’re doing, and not what we lead with anywhere.